controlplane/eks/api/v1beta1/conversion.go

```diff
@@ -42,6 +50,9 @@ func (r *AWSManagedControlPlane) ConvertFrom(srcRaw conversion.Hub) error {
 	if err := Convert_v1beta2_AWSManagedControlPlane_To_v1beta1_AWSManagedControlPlane(src, r, nil); err != nil {
 		return err
 	}
+	if err := utilconversion.MarshalData(src, r); err != nil {
```

Comment:
```suggestion

	if r.Spec.DisableVPCCNI != src.Spec.VpcCni.Disable {
		r.Spec.DisableVPCCNI = src.Spec.VpcCni.Disable
	}

	if err := utilconversion.MarshalData(src, r); err != nil {
```

Reply:
Same goes here right? Only I need to add 

> 		r.Spec.DisableVPCCNI = src.Spec.VpcCni.Disable

Reply:
Yeah

Reply:
Done